omniscripts_benchmarks/hm_fashion_recs/candidates.py

```diff
@@ -61,9 +63,29 @@ def create_candidates_repurchase(
             ["user", "item", "week", "day"]
         ].drop_duplicates(ignore_index=True)
 
-        gr_day = tr.groupby(["user", "item"])["day"].min().reset_index(name="day")
-        gr_week = tr.groupby(["user", "item"])["week"].min().reset_index(name="week")
-        gr_volume = tr.groupby(["user", "item"]).size().reset_index(name="volume")
+        tr = modin_fix(tr)
+
+        # Experimental speedup for modin
+        if check_experimental(modin_exp):
+            gr_day = (
+                tr.groupby(["user", "item"])[["day"]]
+                .min(**exp_grp_kwargs)
+                .squeeze(axis=1)
+                .reset_index(name="day")
+            )
+            gr_week = (
+                tr.groupby(["user", "item"])[["week"]]
+                .min(**exp_grp_kwargs)
+                .squeeze(axis=1)
+                .reset_index(name="week")
+            )
+            gr_volume = (
+                tr.groupby(["user", "item"]).size(**exp_grp_kwargs).reset_index(name="volume")
+            )
```

Comment:
If we're willing to run this on modin's master we would need to set the special env variable instead. 
```suggestion
            modin.config.ExperimentalGroupbyImpl.put(True)
            gr_day = (
                tr.groupby(["user", "item"])[["day"]]
                .min()
                .squeeze(axis=1)
                .reset_index(name="day")
            )
            gr_week = (
                tr.groupby(["user", "item"])[["week"]]
                .min()
                .squeeze(axis=1)
                .reset_index(name="week")
            )
            gr_volume = (
                tr.groupby(["user", "item"]).size().reset_index(name="volume")
            )
            modin.config.ExperimentalGroupbyImpl.put(False)
```